RoosterPlanner.Api/Controllers/ParticipationsController.cs

```diff
@@ -0,0 +1,217 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Claims;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using RoosterPlanner.Api.Models;
+using RoosterPlanner.Common;
+using RoosterPlanner.Models;
+using RoosterPlanner.Service;
+using RoosterPlanner.Service.DataModels;
+
+namespace RoosterPlanner.Api.Controllers
+{
+    [Authorize]
+    [Route("api/[controller]")]
+    [ApiController]
+    public class ParticipationController : ControllerBase
+    {
+        private readonly ILogger logger;
+        private readonly IParticipationService participationService;
+
+        public ParticipationController(ILogger logger, IParticipationService participationService)
+        {
+            this.logger = logger;
+            this.participationService = participationService;
+        }
+
+        [HttpGet]
+        public async Task<ActionResult> GetUserParticipation(Guid personGuid)
+        {
+            try
+            {
+                if (personGuid == Guid.Empty) return BadRequest("No valid id.");
```

Comment:
Ik zou deze regel boven de try/catch zetten. Hier kan weinig aan fout gaan waardoor je een catch nodig hebt en vooral omdat ern voor een try/catch wat data op de stack wordt gezet. Dat is hier nog niet nodig... (Kan zijn dat er hier bij de if al uit de methode wordt gegaan)